### PR TITLE
Persian (Farsi) translations

### PR DESCRIPTION
**Pull request by alisabzevari**
Related to #4,

This PR adds Persian translations.

### CODE REVIEW

**Comment by aatishb** [comment]
Hi @alisabzevari, thanks for working on this! I'll go ahead and merge the live version into this branch, and will let you know if there are any steps remaining.

File: index-fa.html [hunk 1]
```diff
@@ -0,0 +1,940 @@
...
+          هرچه ویروس بیشتری از شخصی به شخص دیگر منتقل شود شانس ابتلا به بیماری بیشتر می‌شود. هم چنین، در افرادی که بیمار
+          می‌شوند، کسانی که در معرض تعداد بیشتری از ویروس بوده‌اند
+          <a href="https://www.nytimes.com/2020/05/29/health/coronavirus-transmission-dose.html">علائم شدیدتری</a>
+          بروز می‌دهند.
+        </p>
+
+      </div>
+
+      <anim-with-caption :mask1="false" :mask2="false" contagiousperson="Contagious Person"
+        susceptibleperson="Susceptible Person">
```
Comment (aatishb) [inline index-fa.html:139]: add translation

File: index-fa.html [hunk 2]
```diff
@@ -0,0 +1,940 @@
...
+        <p>
+          فرض کنید که یک فرد ناقل و یک فرد سالم در محیطی با هم تماس دارند و فرد ناقل از ماسکی استفاده می‌کند که صد درصد
+          موثر نیست و فقط ۵۰٪ کارایی دارد. منظور از «کارایی ۵۰٪» این است که استفاده از این ماسک شانس انتقال بیماری به
+          فرد سالمی را که در نزدیکی فرد ناقل قرار دارد نصف می‌کند. در این شرایط خطر انتقال ویروس نصف میشود.
+        </p>
+
+      </div>
+
+      <anim-with-caption :mask1="true" :mask2="false" :eout="0.5" contagiousperson="Contagious Person"
+        susceptibleperson="Susceptible Person">
```
Comment (aatishb) [inline index-fa.html:182]: add translation

File: index-fa.html [hunk 3]
```diff
@@ -0,0 +1,940 @@
...
+          دارد. برای اینکه موضوع ساده‌تر شود، فرض کنید که ماسک فرضی مورد نظر ما از هر دو طرف به یک اندازه
+          مؤثر است.
+        </p>
+
+        <p>در این صورت، احتمال بیمار شدن فرد سالم از این مسیر هم ۵۰٪ است. </p>
+
+      </div>
+
+      <anim-with-caption :mask1="false" :mask2="true" :ein="0.5" contagiousperson="Contagious Person"
+        susceptibleperson="Susceptible Person">
```
Comment (aatishb) [inline index-fa.html:205]: add translation

File: index-fa.html [hunk 4]
```diff
@@ -0,0 +1,940 @@
...
+        <p>
+          در این حالت، ماسک فرد ناقل خطر انتقال ویروس به محیط را نصف می‌کند، و ماسک فرد سالم هم خطر انتقال ویروس از محیط
+          را نصف می‌کند. بنابراین، وقتی هر دو نفر از ماسک استفاده کنند، ریسک انتقال بیماری نصف از نصف می‌شود. (یا به
+          عبارتی، ۲۵٪ زمانی که هیچ کدام از ماسک استفاده نمی‌کردند). این یعنی احتمال انتقال بیماری ۷۵٪ کاهش پیدا می‌کند.
+        </p>
+
+      </div>
+
+      <anim-with-caption :mask1="true" :mask2="true" :eout="0.5" :ein="0.5" contagiousperson="Contagious Person"
+        susceptibleperson="Susceptible Person">
```
Comment (aatishb) [inline index-fa.html:225]: add translation

File: index-fa.html [hunk 5]
```diff
@@ -0,0 +1,940 @@
...
+            <slider v-model="eout" :min="0" :max="0.99" :step="0.01"></slider>
+          </label>
+
+          <label class="row half bigsplash">
+            <div>کارایی ماسک هنگام دم: <span class="dotted">{{convertToPercent(ein)}}</span>%</div>
+            <slider v-model="ein" :min="0" :max="0.99" :step="0.01"></slider>
+          </label>
+
+          <graph :traces="graph3Traces(
+        '% Infected'
```
Comment (aatishb) [inline index-fa.html:771]: add translation

**Review by aatishb — COMMENTED** [review]
Hi @alisabzevari thanks for working on this, I love the typography! Could you add in these few remaining translations, that should bring us to 100%.

**Comment by aatishb** [comment]
Tagging #4

**Comment by alisabzevari** [comment]
I have addressed the comments. Please take another look.

**Review by aatishb — APPROVED** [review]

**Comment by aatishb** [comment]
@alisabzevari thanks so much for your work on this! Merged.

**Comment by aatishb** [comment]
@all-contributors add @alisabzevari for translation

**Pull request merged**